readthedocs/templates/base.html

```diff
@@ -174,6 +174,9 @@ <h4>{% trans 'Resources' %}</h4>
             <li>
               <a href="https://docs.readthedocs.io">{% trans 'Documentation' %}</a>
```

Comment:
Maybe remove this or the Contributing link, so we don't overwhelm this list? Contributing is probably a good one to go, since it can be confused w/ donating.

Reply:
Good call, will remove it.